### PR TITLE
Fix tinker's compat

### PR DESCRIPTION
**Pull request by vroni92**
Closes #
Closes #
Closes #

# Changes proposed in this pull request:
- Actually check if tinker's is present before trying to use its methods
-
-

Review please

(sorry for this derp)

### CODE REVIEW

**Review by ravenbuilder934 — DISMISSED** [review]

File: src/api/java/com/minecolonies/api/compatibility/CompatibilityManager.java [hunk 1]
```diff
@@ -721,5 +724,10 @@ private void discoverModCompat()
         {
             Compatibility.beeHiveCompat = new ResourcefulBeesCompat();
         }
+        if (ModList.get().isLoaded("tconstruct"))
+        {
+            Compatibility.tinkersCompat = new TinkersWeaponHelper();
+            Compatibility.tinkersSlimeCompat = new SlimeTreeCheck();
```
Comment (someaddons) [inline src/api/java/com/minecolonies/api/compatibility/CompatibilityManager.java:727]: you probably could just do the same with the toolbroken here

Comment (vroni92) [reply src/api/java/com/minecolonies/api/compatibility/CompatibilityManager.java:727]: That one doesn't have a super class yet though. But I could either create one or just relocate the method to the WeaponProxy/Helper

Comment (someaddons) [reply src/api/java/com/minecolonies/api/compatibility/CompatibilityManager.java:727]: ye I guess you could just make it a toolHelper, its similar enoguh

Comment (someaddons) [reply src/api/java/com/minecolonies/api/compatibility/CompatibilityManager.java:727]: like rename the weaponhelper to toolhelper and throw it in there

**Review by someaddons — APPROVED** [review]

**Pull request merged**